.circleci/test-deploy.yml

```
@@ -20,17 +20,39 @@ release-filters: &release-filters
+
+jobs:
+  megalinter-test-deploy:
+    steps:
       - megalinter/megalinter:
+          name: ml
           git_base_branch: master
           persist_megalinter_reports: true
           context:
             - Main Context
           filters: *filters
+      -  megalinter/megalinter:
```

yamllint (1.35.1)

[warning] too many spaces after hyphen

(hyphens)